hpx/lcos/local/packaged_continuation.hpp

```diff
@@ -376,8 +422,9 @@ namespace hpx { namespace lcos { namespace detail
             }
 
             hpx::intrusive_ptr<continuation> this_(this);
-            hpx::util::thread_description desc(f_,
-                "hpx::parallel::execution::parallel_executor::post");
+            char const* annotation = hpx::traits::get_function_annotation<
+                function_type>::call(f_);
```

Comment:
What will be the fallback in this case (i.e. no specialization for `get_function_annotation` is found?

Reply:
The default is to return "<unknown>" or something of that kind. I suppose I'll need to check and see what appears in the traces.